types/win-ca/api.ts

```diff
@@ -0,0 +1,4 @@
+// tslint:disable-next-line: no-relative-import-in-test
```

Comment:
why this file is regular `ts` file, instead of being e.g.`api/index.d.ts` or `api.d.ts`?

Reply:
Ups, this is why I had the lint error... Simply didn't realize that the culprit was the extension...